puppet/modules/scaleway/manifests/cron.pp

```diff
@@ -11,7 +11,7 @@
     owner   => 'root',
     group   => 'root',
     mode    => '0700', #root only, contains API keys
-    content => "api_key: <%= @api_key -%>\norg_id: <%= @org_id  -%>\n",
```

Comment:
Or `"api_key: ${api_key}\norg_id: ${org_id}\n"`

Reply:
That's much better, thanks!